src/app/dataset/depublication/depublication.component.html

```diff
@@ -137,12 +139,14 @@
     ({{ 'depublicationCompleteCountSuffix' | translate }})
   </span>
 
-  <ng-container *ngFor="let record of depublicationData">
-    <app-depublication-row
-      #dataWrapper
-      [record]="record"
-      (checkEvents)="processCheckEvent($event)"
-    ></app-depublication-row>
-    <ng-content *ngTemplateOutlet="dataWrapper.depublicationTemplate"></ng-content>
+  <ng-container *ngIf="!depublicationComplete">
```

Comment:
It looks like the table content becomes invisible if a depublication finishes. Is that what you are doing here? I don't think that's right.

Reply:
I've removed it.